### ISSUE TITLE
https://dart-current-results.web.app "Show latest failures" button leads to blank page

### ISSUE THREAD

**Issue by srawlins** [dart-lang/sdk#53447]
On this page: https://dart-current-results.web.app/#/filter=pkg/dartdev/test/commands/doc_test&showAll, there is a button on the right with hover text "Show latest failures", which opens a new tab at https://dart-current-results.web.app/#showLatestFailures=false&test=pkg/dartdev/test/commands/doc_test. But this page is more or less blank. It starts on a "Failures" tab which just lists instructions about how to write a query.

CC @whesse I hope I'm filing this issue in the right place... let me know if I can help.

**Comment by whesse** [comment]
Yes, this is a failure.  [https://dart-current-results.web.app](https://dart-current-results.web.app/#/filter=pkg/dartdev/test/commands/doc_test&showAll) used to be in a subdirectory at the site [https://dart-ci.firebaseapp.com/](https://dart-ci.firebaseapp.com/), so it's link worked as a relative link.

I'll change this link to one that includes the proper host name.  Fortunately, I have a request for more useful links to be added, so I can include this fix in them.

**Comment by athomas** [comment]
Thanks, yes, there's a bug in grpc we need to fix that causes the server to crash. @whesse is working on a fix.
